boutiques_descriptors/deepprep_24_1_2.json

```diff
@@ -0,0 +1,116 @@
+{
+    "name": "DeepPrep",
```

Comment:
@prioux not sure, do we have some convention to add a suffix SingleSubjects or BidsSubject to tool name or that practice is deprecated?